src/bot/scripts/card.js

```diff
@@ -27,18 +26,21 @@ let frameTops = null;
 let typeIcons = null;
 let miscIcons = null;
 
-function removeUnderscores(string) {
+const removeUnderscores = string => {
   string = string.replace(/_/g, " ").replace(/\w\S*/g, function (txt) {
     return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
   });
   return string;
-}
+};
 
 let cache = {
   date: 2,
```

Comment:
regarding the default values here;
- set the empty strings to null instead
- why the 2?, just use a 0